### DISCUSSION TITLE
How to install version 1.7 beta

### DISCUSSION THREAD

**Discussion by snden** [vkbo/novelWriter#1145] [Help]
I would like to install version 1.7 beta. When I run
```
sudo add-apt-repository ppa:vkbo/novelwriter-pre
sudo apt update && sudo apt install novelwriter
```
Version v1.6-rc1 is installed. How do I install the 1.7 beta version?
Or is appimage version 1.7 beta available?
- - -
Linux Mint 19.2, Xfce

**Comment by vkbo** [comment]
The appimages will hopefully be available from the 1.7 RC 1 release and on. They were added after the 1.7 Beta release.

I'm not sure why you're getting a 1.6 reelase from the novelwriter-pre repo. As far as I can see, you should be getting one of the snapshot releases: https://launchpad.net/~vkbo/+archive/ubuntu/novelwriter-pre

**Reply by vkbo** [reply]
Note that I've renamed release 1.7 to 2.0 due to the significant changes I've been making. It makes more sense. See #1144.

I've also uploaded a new snapshot release to novelwriter-pre, so you should now be getting a 2.0-b1 instead.

**Reply by vkbo** [reply]
I also think I know why you're getting an older version from the repo. You are probably getting the version for Ubuntu 21.04, but 21.04 is deprecated, so there are no updates for it. Launchpad will not let me uploaded new releases for no longer supported versions of Ubuntu.

New releases are currently being uploaded for Ubuntu 20.04, 22.04 and 22.10 only.

**Reply by snden** [reply]
Thanks for the help, it was caused by the linux version. I installed Linux Mint 20.3, which is based on Ubuntu Focal, on another PC, and from vkbo/novelwriter-pre, version 1.7beta was already installed.
